bindings/java/src/org/sleuthkit/datamodel/AggregateScoresChangedEvent.java

```diff
@@ -1,54 +0,0 @@
-/*
```

Comment:
Will any of these refactorings or renames affect downstream clients of this code?